package.json

```
@@ -12,12 +12,12 @@
     "test": "yarn run lint && yarn run lint:prettier && yarn run test:unit && yarn run typecheck",
     "test:unit": "jest",
     "lint": "eslint --max-warnings=0 '{src,examples}/**/*.{ts,tsx,js}'",
-    "lint:prettier": "prettier './**/*.js' './**/*.css' './**/*.md' --list-different",
+    "lint:prettier": "prettier './**/*.js' './**/*.ts' './**/*.css' './**/*.md' --list-different",
```

Whoops good catch. I think we need `tsx` too:

```suggestion
    "lint:prettier": "prettier './**/*.js' './**/*.ts' './**/*.tsx' './**/*.css' './**/*.md' --list-different",
```